superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -555,6 +557,9 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
             return;
           }
 
+          if (data.type === 'Alert') addSuccessToast(t('Alert updated'));
+          else addSuccessToast('Report updated');
```

Comment:
you may want to translate this one, too. 
Also, I think you can simplify this by using the data.type in the toast... something like: 
```
addSuccessToast(t(`${data.type} updated`));
```
Assuming that the data.type for reports is "Report"?

Reply:
Gotcha! will do!